app/models/product.rb

```diff
@@ -155,6 +163,12 @@ def groups
     permissions(:group)
   end
 
+  def active_users
+    @active_users ||= `ps -o uid= -p $(pgrep -f '^Passenger .*#{Regexp.quote(router.path.realdirpath.to_s)}') 2> /dev/null | sort | uniq`.split.map(&:to_i).map do |id|
+      OodSupport::User.new id
```

Comment:
That is true, but not sure how the `ps` command could give us a `uid` that isn't an actual user.